notebooks/01 - Introduction.ipynb

```diff
@@ -32,7 +32,7 @@
    "outputs": [],
    "source": [
     "from unfolded.map_sdk import UnfoldedMap\n",
-    "unfolded_map = UnfoldedMap()"
+    "unfolded_map = UnfoldedMap(height=500)"
```

Comment:
Can we make this an internal default so we don't need to change all the notebooks?

Reply:
I would recommend we use 600 or 640px for that default height setting, based on some embedded Unfolded Studio cell output experiments I've done with JS notebooks.